resources/views/components/slideover.blade.php

```diff
@@ -0,0 +1,14 @@
+@props(['id', 'title' => ''])
+<input type="checkbox" v-bind:id="{{ $id }}" class="peer hidden"/>
+<label v-bind:for="{{ $id }}" class="pointer-events-none fixed inset-0 z-50 cursor-pointer bg-ct-neutral/30 opacity-0 transition peer-checked:pointer-events-auto peer-checked:opacity-100"></label>
+<div class="pointer-events-none fixed top-0 -right-full z-50 flex h-screen w-full max-w-md flex-col bg-white transition-[right] duration-300 peer-checked:pointer-events-auto peer-checked:right-0">
```

Comment:
why use h-screen when the element has fixed, isn't inset-y-0 the same way to achieve that plus on mobile the h-screen class causes it actually to be higher than your screen height because of the url bar. Can you check that for me?